src/Sulu/Bundle/AdminBundle/Resources/js/views/Form/toolbarActions/CopyLocaleToolbarAction.js

```diff
@@ -2,18 +2,48 @@
 import React from 'react';
 import {action, observable} from 'mobx';
 import jexl from 'jexl';
+import log from 'loglevel';
 import Checkbox from '../../../components/Checkbox';
 import Dialog from '../../../components/Dialog';
 import ResourceRequester from '../../../services/ResourceRequester';
 import {translate} from '../../../utils/Translator';
-import AbstractFormToolbarAction from './AbstractFormToolbarAction';
+import {ResourceFormStore} from '../../../containers/Form';
+import Form from '../Form';
+import Router from '../../../services/Router';
 import copyLocaleActionStyles from './copyLocaleAction.scss';
+import AbstractFormToolbarAction from './AbstractFormToolbarAction';
 
 export default class CopyLocaleToolbarAction extends AbstractFormToolbarAction {
     @observable showCopyLocaleDialog = false;
     @observable selectedLocales: Array<string> = [];
     @observable copying: boolean = false;
 
+    constructor(
+        resourceFormStore: ResourceFormStore,
+        form: Form,
+        router: Router,
+        locales: ?Array<string>,
+        options: {[key: string]: mixed}
+    ) {
+        const {
+            display_condition: displayCondition,
+            visible_condition: visibleCondition,
+        } = options;
+
+        if (displayCondition) {
+            log.warn(
```

Comment:
We also have to think about how to find these deprecations later when we want to remove them... Searching for `log.warn` might be a little bit a hassle... Maybe we should use the `deprecated` annotation of JS Doc? Should be easy to search for afterwards :slightly_smiling_face:

This of course applies to all deprecations you have done in here.

Reply:
conducted a short google search and did not find anything better than the `@deprecated` JSDoc. but we are not deprecating a function here.. 

where would you add the `@deprecated` tag? as a comment in the conde?

Reply:
That's the closest thing I can think of. If you add a message we know what to do, when we search for the `@deprecated` tag in the entire project before the 3.0 release.

Reply:
done 🙂